cma/transformations.py

```diff
@@ -1029,7 +1029,7 @@ def pheno(self, x, into_bounds=None, copy=True,
         input_type = type(x)
         if into_bounds is None:
             into_bounds = (lambda x, copy=False:
-                                x if not copy else array(x, copy=copy))
+                                x if not copy else np.asarray(x, copy=copy))
```

Comment:
This looks like a bug: `np.asarray` doesn't accept a `copy` argument?

Reply:
Nice catch! I'll change this to `array(x, copy=True)` to make it clear that it's only called when `copy=True`.